webapp/dev_settings.py

```diff
@@ -1,5 +1,4 @@
-from .settings import *
```

Comment:
Ah I wasn't aware of that recommendation in PEP8.

Looking into it, the wording around this has [softened recently](https://www.python.org/dev/peps/pep-0008/#imports) because imports have been improved to remove the possible dangers of relative imports:

> Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path )
> ...
> However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose

There's a longer-form [discussion about this](http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module#answer-16748366) where they mention why there is no longer a strong reason for discouraging relative imports.

Although this module definitely doesn't constitute a "complex package layout", it seems to me that considering all the wider context, the decision to use explicit relative imports is now more-or-less an individual choice (implicit relative imports should never be used), and to my mind the arguments for relative imports in this case slightly outweigh any arguments for absolute ones.

However, I think we're only in the process of the advice on explicit relative imports changing, rather than quite there yet, so I'm happy to leave these imports absolute as you've done them for now, at least until we're solidly on Python 3 throughout all our projects but I think we should remain open to revising advice on that in the future.